third_party/silabs/silabs_board.gni

```diff
@@ -106,27 +107,31 @@ if (silabs_board == "BRD4304A") {
   disable_lcd = true
   show_qr_code = false
   wifi_soc = true
+  silabs_board_lower = "brd4325b"
```

Comment:
I believe there is a way in gn to convert a string to lower instead of adding a new var

Reply:
did not find it, for now. will do it in the upcoming PRs once we find it.

Reply:
ok I will change it when I get the chance